files/scripts/syncd.sh

```diff
@@ -150,6 +151,12 @@ stop() {
         TYPE=cold
     fi
 
+    if [[ x$sonic_asic_platform == x"mellanox" ]] && [[ x$TYPE == x"cold" ]]; then
```

Comment:
Note that TYPE is set according to /proc/cmdline.
In the flow like this:
```
1. sudo warm-reboot
2. # wait for warmboot-finalaizer to finish and disable services WR mode
3. sudo systemctl restart swss # which restarts syncd
```
In such flow x$TYPE == x"fastfast" but obviously such service restart was meant to be cold

Reply:
@stephenxs Oh, yes. We have 3 variables here representing almost the same thing (BOOT_TYPE, WARM_BOOT, TYPE) which confused me